packages/esm-service-queues-app/src/config-schema.ts

```diff
@@ -196,21 +218,150 @@ export const configSchema = {
     _default: '',
     _description: 'Custom URL to load default facility if it is not in the session',
   },
-  tablesConfig: {
-    _type: Type.Object,
-    _description: `Configurations of columns to show for the queue table.
-      Multiple configurations can be provided, each can be applied generally, or to tables 
-      for a particular queue, particular status, or even particular queue+status combination.
-      If multiple configs are defined, the first config with matching appliedTo condition
-      is used.
-      See https://github.com/openmrs/openmrs-esm-patient-management/blob/main/packages/esm-service-queues-app/src/config-schema.ts
-      for full schema definition and example.
-    `,
-    _default: defaultTablesConfig,
+  ...columnConfigSchemaFragment,
```

Comment:
This line is unintentional, right? `columnConfigSchemaFragment` also appears in the `queueTables` Object.

Reply:
This makes it so these properties can be passed at the "global" level or per-column. So if you set `priorityConfigs` at the top level, it will apply to all columns of type `priority` unless overridden. I also added a test for this. This kind of thing is already [in the MCOE config](https://github.com/PIH/openmrs-config-pihsl/blob/ac6f5914cbd0d5c41923903634d128c849bc84f5/configuration/frontend/config.json#L255-L275). However, it presumably wasn't doing anything, because there was no code to actually handle column config elements passed at the top level.

Reply:
Hm... not sure if I like this. If we create a `TableDefinition` with no `appliedTo` criteria, it will also apply globally. When there are multiple `TableDefinition` with matching `appliedTo` criteria, currently we just pick the first matching one. This `global` column config makes the order of precedence more complex. I can imagine there are cases where this simplifies things, but I think it makes the config (even) harder to understand.

Reply:
Sure, fair enough. I'll remove that.

Reply:
Removed, including the test which tested this behavior.